agent_app_sample_code/utils/file_loading.py

```diff
@@ -127,7 +121,7 @@ def load_uc_volume_to_delta_table(
         display_markdown(
             f"### {num_errors} documents had parse errors. Please review.", raw=True
         )
-        display(errors_df)
```

Comment:
display() doesn't work in unit tests, asked EDA folks about it in https://databricks.slack.com/archives/C010N79S8GZ/p1727367703584909

Reply:
thats unfortunate. if we're not writing tests that actually look at what is displayed, can we just monkeypatch this out?